go.mod

```
@@ -55,12 +55,12 @@ require (
 	github.com/stretchr/testify v1.8.4
 	github.com/vultr/govultr/v2 v2.17.2
 	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.42.0
```

```suggestion
	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.44.0
```
Should this dependency not also be updated?

---

Yes, good catch!